Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -103,7 +103,7 @@ class SearchViewController: SiteTableViewController, KeyboardHelperDelegate {
         searchEngineScrollViewContent.subviews.map({ $0.removeFromSuperview() })
 
         var leftEdge = searchEngineScrollViewContent.snp_left
-        for engine in searchEngines.enabledEngines {
+        for engine in searchEngines.quickSearchEngines {
```

Comment:
Note that as of PR #191, we already weed out the default engine when building this list, so it looks like you can remove that bit when you rebase this.